server/src/main/java/com/cloud/server/ManagementServerImpl.java

```diff
@@ -2128,7 +2128,7 @@ private void abstractDataStoreClustersList(List<StoragePool> storagePools, List<
             _accountMgr.buildACLSearchBuilder(sb, domainId, isRecursive, permittedAccounts, listProjectResourcesCriteria);
         }
 
-        buildParameters(sb, cmd);
+        buildParameters(sb, cmd, true);
```

Comment:
@weizhouapache If _isAllocated_ is hardcoded in both the _buildParameters()_ calls, where exactly we use the 'allocatedonly' parameter from listPublicIpAddresses API cmd.

Reply:
@sureshanaparti good point.
isAllocated is used in line 2136.
I updated this pr to fix an issue with shared networks.